src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/StreamingClientManager.java

```diff
@@ -52,6 +56,17 @@ protected StreamingClientManager() {
     this.clientCount = -1; // will be incremented when a client is created
   }
 
+  /** Resets existing streaming clients and get a new client manager used in testing. */
+  public static StreamingClientManager resetAndGetEmptyStreamingClientManager() {
```

Comment:
code coverage? this file has 100% but I don't see a corresponding test in StreamingClientManagerTest

Reply:
Ack, will do!